lineman/app/components/thread_preview_collection/thread_preview_collection.haml

```diff
@@ -1,3 +1,3 @@
 .thread-previews
-  .blank{ng-repeat: 'thread in query.threads() track by thread.id | limitTo: limit'}
+  .blank{ng-repeat: 'thread in query.threads() | orderBy:importance track by thread.key | limitTo: limit'}
```

Comment:
This also fixed a dumb bug we were having (apparantly the ids of discussions weren't being read as unique for some reason... still not sure exactly why.)